content/src/content/jcr_root/apps/core/wcm/components/commons/site/clientlibs/utils/js/utils.js

```diff
@@ -77,9 +77,11 @@
             return transformedProperties;
         };
 
+        document.dispatchEvent(new CustomEvent('core.wcm.components.commons.site.utils.loaded'));
```

Comment:
This will immediately call the subscriber of the event, before the object returned below actually gets assigned to the global variable. 

The event should be fired outside of this function.